examples/dotnet/README.md

```diff
@@ -9,13 +9,28 @@
 * [Apache.Geode.Examples.AuthInitialize](Apache.Geode.Examples.Cache/README.md)
 * [Apache.Geode.Examples.Cache](Apache.Geode.Examples.Cache/README.md)
 
-## Using
-1) Use cmake to generate the .sln and .vcxproj files for the Examples
-```
-    cd examples\dotnet
+## Building the Examples
+
+1. Copy the `examples` directory from the native client installation folder to a folder in your workspace.
+
+1. Use cmake to generate the .sln and .vcxproj files for the examples:
+
+    ```
+    cd workspace\examples\dotnet
     mkdir build
     cd build
-    cmake .. -G"Visual Studio 14 2015 Win64" -DGEODE_ASSEMBLY=C:\path\to\Apache.Geode.dll -Thost=x64
-```
-2) Open `Apache.Geode.Examples.sln` in [Visual Studio 2015](https://www.visualstudio.com/)
-or newer.
+    cmake .. -G"Visual Studio 14 2015 Win64" -DGeodeNative_ROOT="c:\Program Files\nativeclient"
```

Comment:
yes

Reply:
if the example development should be in a separate workspace maybe a separate repository etc. might be in order....

Reply:
For what it's worth, the installation is useless for any cloud native style web application where you have to include all your dependencies with your web application when you deploy it to some cloud.  I guess that is making an assumption about a use case of Geode Native, but I would end up ignoring most of these instructions when including this as a dependency in my web application.

Reply:
@moleske the target here isn't "cloud native". Every cloud has different requirements so documenting them is would be exhausting. We document an opinionated step that will get you going on your development environment. Taking that to production is your task.

Reply:
I guess I was just saying that I wouldn't follow this still for my development environment since it wouldn't match what production would look like.  But in the grand scheme of things, you are right going to production is the consumer's task.  Just trying to get ideas out that could make it easier to keep your development and production environments more similar.